File: Exercicio4.c
#include <stdio.h>

int main(void) {
  int num, horas;
  float sal;
  
  scanf("%d", &num);
  scanf("%d", &horas);
  scanf("%f", &sal);

  sal = (sal*horas);
  
  printf("NUMERO = %d",num);
  printf("\nSALÁRIO = R$ %.2f",sal);
  return 0;
}

File: Exercicio4.cs
using System;

class Exercicio4 {
  public static void Main (string[] args) {
    int num, horas;
    float sal;

    num = int.Parse(Console.ReadLine());
    horas = int.Parse(Console.ReadLine());
    sal = int.Parse(Console.ReadLine());

    sal = (sal*horas);
    
    Console.WriteLine ($"NÚMERO = {num}");
    Console.WriteLine ($"SALÁRIO = {sal:F2}");
  }
}
